src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js

```diff
@@ -149,7 +143,7 @@ function addRulesAndVariablesFromProgramIndicators(cachedProgramIndicators: Arra
     });
 }
 
-function sortPrograms(programs: Array<Program>) {
+function sortPrograms(programs: Array<EventProgram | TrackerProgram>) {
```

Comment:
Also not sure I understand the difference between `Program` and `EventProgram | TrackerProgram`. I changed it because flow was complaining that some functions dont exist in Program but exist on `EventProgram` or `TrackerProgram`. 

Perhaps you could explain the differences to me if you got a minute.